test/test_little_dict.jl

```diff
@@ -489,6 +489,7 @@ using OrderedCollections: FrozenLittleDict, UnfrozenLittleDict
         @test d_wide isa LittleDict{Symbol, Number}
 
         @test_throws MethodError LittleDict{Char,Char}(:x => 1)
+        @test isequal(LittleDict{Int,Int}(), LittleDict{AbstractString,AbstractString}())
```

Comment:
this is unrelete the that issue, but you can put it further up in this file with the other `isequal` tests

Reply:
Ohh! It's already there :sweat_smile:

Reply:
Just now, I saw your comment in the corresponding testset! Yes, it's sort of a descriptive comparison, rather than absolute equality.